src/higley_lab_to_nwb/benisty_2022/benisty_2022_convert_session.py

```diff
@@ -17,27 +17,17 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
         output_dir_path = output_dir_path / "nwb_stub"
     output_dir_path.mkdir(parents=True, exist_ok=True)
 
-    session_id = "subject_identifier_usually"
+    session_id = "11222019_grabAM05_spont"
     nwbfile_path = output_dir_path / f"{session_id}.nwb"
 
     source_data = dict()
     conversion_options = dict()
 
-    # Add Recording
-    source_data.update(dict(Recording=dict()))
-    conversion_options.update(dict(Recording=dict()))
-
-    # Add LFP
-    source_data.update(dict(LFP=dict()))
-    conversion_options.update(dict(LFP=dict()))
-
-    # Add Sorting
-    source_data.update(dict(Sorting=dict()))
-    conversion_options.update(dict(Sorting=dict()))
-
-    # Add Behavior
-    source_data.update(dict(Behavior=dict()))
-    conversion_options.update(dict(Behavior=dict()))
+    # Add Wheel signal
+    file_path = str(data_dir_path / session_id / f"{session_id}_spike2.smrx")
+    stream_id = '4'
+    source_data.update(dict(Wheel=dict(file_path=file_path, stream_id=stream_id, es_key="Wheel")))
```

Comment:
I'd keep the `ElectricalSeries` prefix
```suggestion
    source_data.update(dict(Wheel=dict(file_path=file_path, stream_id=stream_id, es_key="ElectricalSeriesWheel")))
```